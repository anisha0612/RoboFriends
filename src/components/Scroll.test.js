import React from "react";
import { create } from "react-test-renderer";
import Scroll from "./Scroll";

it("Check if the Scroll component is rendered", () => {
  expect(create(<Scroll />).toJSON()).toMatchSnapshot();
});
