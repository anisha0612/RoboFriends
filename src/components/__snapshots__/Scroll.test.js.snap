// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if the Scroll component is rendered 1`] = `
<div
  style={
    Object {
      "height": "700px",
      "overflowY": "scroll",
    }
  }
/>
`;
